### ISSUE TITLE
Could not decrypt pak files

### ISSUE THREAD

**Issue by Dailowo** [CutFlame/MCDSaveEdit#73]
I used the Storepatcher v2 to get those .pak files and mcd save editor doesnt open them, any ideas on how i can save this?
![image](https://user-images.githubusercontent.com/88281532/127775978-b2c45b93-363c-48dc-a7b6-330e7f475e69.png)
![image](https://user-images.githubusercontent.com/88281532/127775981-6584f5af-8780-41e6-91ec-4e858d0f6a79.png)

**Comment by Thana1995** [comment]
Check the Issue before. Its because the newest Version here is outdated.
I had the same issue.
The beta patreon works just fine

**Comment by grim9x8** [comment]
> Check the Issue before. Its because the newest Version here is outdated.
> I had the same issue.
> The beta on patreon works just fine

is this still the case im still getting the error and have the latest version, and the latest i see on the patreon is the public one.

**Comment by regnirok** [comment]
> > Check the Issue before. Its because the newest Version here is outdated.
> > I had the same issue.
> > The beta on patreon works just fine
> 
> is this still the case im still getting the error and have the latest version, and the latest i see on the patreon is the public one.

Yeah, a new version of MC:D(1.10.3.0) was released two days ago.  You'll run into this issue until MCDSE is updated to support this.
